sys-apps/dbus/dbus-1.16.2.recipe

```diff
@@ -103,19 +110,17 @@ defineDebugInfoPackage dbus$secondaryArchSuffix \
 
 BUILD()
 {
-	runConfigure --omit-dirs binDir ./configure \
-		--with-system-pid-file=/var/run/dbus/pid \
-		--with-system-socket=/var/run/dbus/system_bus_socket \
-		--bindir=$prefix/bin \
-		--disable-static
-	make $jobArgs
+	cmake -B build -S . -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		$cmakeDirArgs \
```

Comment:
This one is probably missing `-DCMAKE_INSTALL_DOCDIR=$documentationDir/packages/dbus`, otherwise it uses `.../dbus_x86` on 32 bit. (We have to do it in the recipes, because https://github.com/haikuports/haikuporter/pull/337 doesn't work correctly in some cases.)

Reply:
I guess the answer is in the patch, they used `-DCMAKE_INSTALL_DATADIR/doc/dbus`, so even setting this to the above wouldn't help. :)

Reply:
No, I meant in addition to that, overriding the default value set by `$cmakeDirArgs`. On primary architectures, this is the same as `$docDir` (the default), but on secondary architectures, it removes the secondaryArchSuffix from the value of docDir. This is similar to overriding `CMAKE_INSTALL_BINDIR` for binaries, as you noted recently (I think in IRC).

Reply:
I think I'm confused now, wasn't that changed dropped?

Reply:
Yes, that's why we have to adjust the affected recipes instead (i.e. all recipes with an any-arch doc package that are built only on secondaryArch on 32 bit). The haikuporter change would have been a shortcut to avoid that and let haikuporter do it automatically, but that was too "broad" and would also change packages where we don't want the change.

Reply:
Have to check later on 32bit then. :)